src/utils.ts

```diff
@@ -500,3 +501,31 @@ export function getFileExtension(pathAsString: string) : string | undefined {
 	}
 	return undefined;
 }
+
+export function getAppendRegisteredSettingFromEnv() : string | undefined {
+	console.log(process.env);
```

Comment:
i think that the console.log shoudl be avoided in production code in general.
Also, printing the Process environment seems to be a security risk (for instance if there are tokens set in environments)